src/alternating_update.jl

```diff
@@ -85,6 +85,7 @@ function alternating_update(
         mindim=mindim[sweep],
         cutoff=cutoff[sweep],
         noise=noise[sweep],
+        outputlevel=outputlevel,
```

Comment:
```suggestion
        outputlevel,
```
That's a shorthand you can use in more recent versions of Julia. Also you can you move it up above the truncation parameters?

Reply:
sure thing!